.changeset/unlucky-windows-compete.md

```diff
@@ -0,0 +1,5 @@
+---
+'@hyperlane-xyz/registry': patch
```

Comment:
```suggestion
'@hyperlane-xyz/registry': minor
```

Reply:
Yes, fixed.

Reply:
@JaunePomme I've noticed you fix these by adding new files and creating duplicate changesets. I've fixed it for you a few times but I'd appreciate it if you could just modify the existing file. All you have to do is click the `commit suggest` button on my first comment

Reply:
My bad, I'm used to Gitlab and not Github, and never used the changeset system so I kept creating new ones;
Thank you, I'll keep learning and do my best !